libqalculatenasc/meson.build

```diff
@@ -0,0 +1,10 @@
+libqalculatenasc = static_library(
+    'libqalculatenasc',
+
+    'QalculateNasc.cc',
+    'QalculateNasc.h',
```

Comment:
The header should be autodetected by meson.

Reply:
Does it hurt if it's added explicitly?